src/midi/fluid_midi.c

```diff
@@ -1719,8 +1816,8 @@ new_fluid_player(fluid_synth_t *synth)
     player->end_pedals_disabled = 0;
     player->last_callback_ticks = -1;
     fluid_atomic_int_set(&player->seek_ticks, -1);
-    fluid_player_set_playback_callback(player, fluid_synth_handle_midi_event, synth);
-    fluid_player_set_tick_callback(player, NULL, NULL);
+    fluid_player_set_playback_callback(player, fluid_synth_handle_midi_event, player);
+    fluid_player_set_tick_callback(player, fluid_player_handle_loops, player);
```

Comment:
It should be documented in `fluid_player_set_tick_callback` that loops will no longer work if the user registers his own tick_callback. Which is why this function was invented.